pact-jvm-provider-sbt/build.sbt

```diff
@@ -1,9 +1,15 @@
 sbtPlugin := true
 isSnapshot := true
 
+scalaVersion := "2.12.4"
+organization := "au.com.dius"
+
+val currentVersion = "3.5.11"
+version := currentVersion
+
 libraryDependencies ++= Seq(
-  "au.com.dius" %% "pact-jvm-provider-sbtsupport" % Common.version,
```

Comment:
I think `Common.version` was broken after the removal of `project/Build.scala`. For now I am setting the project version here, but it needs to be manually updated on every release.